package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "fakebook",
+  "version": "1.0.0",
+  "description": "A facebook clone built using Node.js, React with hooks and Typescript. From The Odin Project.",
```

Comment:
This project (and the client as well) isn't using typescript. It's using Javascript.

For express _maybe_ this doc would help? Or you can find another that you think would work, https://developer.okta.com/blog/2018/11/15/node-express-typescript

For Typescript with `create-react-app` have a look at https://create-react-app.dev/docs/adding-typescript/